### PR TITLE
WIP: add rudimentary drone.io support, and add latitude/longitude support, fix `staticcheck` errors

### LINKED ISSUE #62: Explicit latitude/longitude settings for weather forecast lookup
If the geocoder API is over limit (see #54) you get an error rather than a forecast.

Handle the case where you know exactly where you are already, and pass in the latitude and longitude directly to the API, e.g. support queries like so:

```
weather --latitude 42.27756 --longitude -83.74088
```

which will give you Ann Arbor, MI, or the same output more or less as

```
weather -l "Ann Arbor, MI"
```

I have a work in progress to do this - the only small hitch so far is that the output routine counts on there being a nice name for the location.

### PR DESCRIPTION
**Pull request by vielmetti**
Do not merge (it's not done yet, there's some more tweaks to the output).

Try to find explicit lat/lon and if you find them, skip the geocoding.
Also introduces a `.drone.yml` file to test this out, using Drone Cloud.
Fix the "staticcheck" errors so the build is green.

This probably should be pulled apart into a couple of separate PRs, and squashed and made all pretty.

Closes #62

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Found some fixes!

_P.S. share your ideas, feedbacks or issues with us at https://github.com/fixmie/feedback (this message will be removed after the beta stage)._

**Comment by vielmetti** [comment]
ok, this passes all the checks. it's really three PRs in one:

- add Drone Cloud support
- fix staticcheck changes
- add lat/lon

probably better to tackle each of those individually rather than merge this whole mess in.

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+kind: pipeline
+name: drone-weather
+
+workspace:
+  base: /go
+  path: src/github.com/vielmetti/weather
```
Comment (vielmetti) [inline .drone.yml]: This should be src/github.com/genuinetools/weather

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+steps:
+  - name: test
+    image: golang:1.11
```
Comment (vielmetti) [inline .drone.yml]: Probably should be golang:1.10 to match the mainline

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      - go get
+      - go build -o weather
+      - chmod +x weather
+  - name: run
+    image: golang:1.11
+    commands:
+      - go get
+      - go build -o weather
+      - chmod +x weather
+      - ./weather --latitude 42.27756 --longitude -83.74088
```
Comment (vielmetti) [inline .drone.yml]: The intent is to test just this new lat/lon change

**Comment by vielmetti** [comment]
While this was well-intentioned, it shouldn't be merged. Closing as `wontfix`.

**Pull request closed without merging**